x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/policy_response.tsx

```diff
@@ -48,15 +49,15 @@ const PolicyResponseConfigAccordion = styled(EuiAccordion)`
 const ResponseActions = memo(
   ({
     actions,
-    actionStatus,
+    responseActions,
   }: {
-    actions: Immutable<Array<keyof HostPolicyResponseActions>>;
-    actionStatus: Partial<HostPolicyResponseActions>;
+    actions: ImmutableArray<string>;
```

Comment:
Use `Immutable<>` (same below in several places). Also, would this change based on my prior suggestion to type up known action names?